drivers/hmis/spec/requests/hmis/update_external_form_submission_spec.rb

```diff
@@ -303,6 +303,100 @@
         end
       end
 
+      context 'when the submission specifies age range, a special-case client attribute' do
+        let!(:submission) do
+          data = {
+            'Client.firstName': 'foobar',
+            'Client.ageRange': '18-24',
+            'Client.dob': '',
+          }.stringify_keys
+          create(:hmis_external_form_submission, raw_data: data, definition: definition)
+        end
+
+        it 'should process the age range onto DOB with low data quality' do
+          expect do
+            response, result = post_graphql(input) { mutation }
+            expect(response.status).to eq(200), result.inspect
+            expect(result.dig('data', 'updateExternalFormSubmission', 'externalFormSubmission', 'status')).to eq('reviewed')
+          end.to change(Hmis::Hud::Client, :count).by(1).
+            and change(Hmis::Hud::Enrollment, :count).by(1)
+
+          submission.reload
+          expect(submission.enrollment.client.dob).to be_between(Date.today - 24.years, Date.today - 18.years)
```

Comment:
To reduce test flakiness, can define a variable `let(:today) { Date.current }` and reference it throughout the tests. Then there aren't failures at midnight. Also reminder on `current` instead of `today`

Reply:
Thanks, sorry

Reply:
all good, thanks!